### ISSUE TITLE
0.9 consumer doesn't add tags or fields

### ISSUE THREAD

**Issue by raviv** [logstash-plugins/logstash-input-kafka#80]
My conf is 
````
  kafka {
    topics => ["eventlog"]
    bootstrap_servers => "localhost:9092"
    tags => ["kafka"]
    add_field => {"foo" => "bar"}
    codec => json {}
  }
````
 
But the tag and field are missing from resulting event.
I'm using beta 4

**Comment by talevy** [comment]
thanks for reporting this!

looks like a simple fix. will be updated shortly for another beta release.

to track its progress, please check out this PR: https://github.com/logstash-plugins/logstash-input-kafka/pull/81

it is a simple fix, but my other reviewer is out in training until later this week, so it may not get in for another few days

**Comment by suyograo** [comment]
Fixed in #81
